copy_edit.py

```diff
@@ -75,14 +76,19 @@ def run(self, edit):
 		if numsels == 0:
 			return
 		
+		if numsels == 1: # To fix TN 7
+			selection_strings = [(("" if selection_strings[0][1] else "\n") # проверка нужна, так как если selection_strings[0][1] == True, то \n уже есть в конце строки
```

Comment:
Any chance you could translate this? I'm not positive what this is fixing.

Reply:
Ok, I have added [commit](https://github.com/adzenith/CopyEdit/pull/8/commits/87b0adc0c46ffa349ce879b42f3aacd8d575b71e) with translation of this comment.
About "what this is fixing" please see test, which was added in this commit (if you wish I can explain what it means, but it's obvious [`DA >‘1’<2` means insert text `12` and select `1`]).